awscli/examples/ssm/add-tags-to-resource.rst

```diff
@@ -1,7 +1,25 @@
-**To add tags to a resource**
+**To add tags to an SSM Document**
 
-This example updates a maintenance window with new tags. There is no output if the command succeeds.
+This example updates an SSM document with new tags. There is no output if the command succeeds.
```

Comment:
```suggestion
The following ``add-tags-to-resource`` example adds a tag to the specified document. ::
```

What do you think of this suggestion? This would help keep the language consistent with the current wording in the [other examples](https://raw.githubusercontent.com/aws/aws-cli/develop/awscli/examples/ssm/add-tags-to-resource.rst).